whereby/sdk
# browser-sdk: Add constrained (mobile) grid to video grid

### Description

Adds the constrained grid version of the video grid to the SDK. Currently the breakpoint for this to trigger is if either height or width of the container is < 500px. The constrained grid displays cells differently (takes up more of the "free" space) by stretching videos, removing border-radius, and displays a different grid layout. 

The constrained grid is enabled by default, but it can be turned off, by passing `enabledConstrainedGrid = false` to the `<VideoGrid />` component. 

**Summary:**



**Related Issue:**



### Testing

1. Build SDK with local-stack URLs
2. Run storybook (`yarn dev`)
3. Test the Video Grid Story
4. Check with different amount of participants (and mute/unmute states)
5. Check with different mobile resolutions
6. Verify that disabling the `enableConstrainedGrid` option (It's a storybook control) turns the constrained grid behaviour off

### Screenshots/GIFs (if applicable)
![Screenshot 2024-08-09 at 14 04 27](https://github.com/user-attachments/assets/dd59e49c-6758-480a-a53d-2c488ed5eb6f)
![Screenshot 2024-08-09 at 14 04 11](https://github.com/user-attachments/assets/5b7753ef-35f3-474e-847b-3b4b6fbd0d8d)



### Checklist

        under the terms of the MIT license.

### Additional Information
